community/meetup/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Knative Community Meetup"
+linkTitle: "Community Meetup"
+weight: 40
```

Comment:
I think you want this to be `weight: 30` so that it collates before https://github.com/knative/docs/blob/master/community/contributing/_index.md

Reply:
done 🎉